src/components/interfaces/HMI_API.xml

```diff
@@ -6023,7 +6042,10 @@
       <description>The instantaneous fuel consumption in microlitres</description>
```

Comment:
We should also add a note in the `fuelLevel` and `fuelLevel_State` descriptions stating that the parameters are deprecated in the MOBILE_API